docs/src/ice_nucleation_plots/KnopfAlpert2013_fig1.jl

```diff
@@ -0,0 +1,167 @@
+import Plots
```

Comment:
Could you 
```suggestion
import Plots as PL
```

and then delete `const PL = Plots` .  And the same for other packages. This is an old Julia syntax and we are trying to move away from it. - It should be hopefully updated in most of the places.